src/index.js

```diff
@@ -14,10 +15,18 @@ if (process.env.NODE_ENV !== "production") {
 }
 
 const mapConfig = parseUriHash(defaultMapConfig);
+var translator = new Translator({
+  persist: false,
+  languages: ["en", "pt"],
+  defaultLanguage: mapConfig.lang,
+  detectLanguage: true,
+  filesLocation: "/i18n",
+});
 
 new LeafletMap(mapConfig);
-new InfoWindow();
+new InfoWindow(translator);
 new TitleDetails();
 new LoadingIndicator();
 
-getData();
+translator.load();
+getData({ lang: mapConfig.lang });
```

Comment:
this is a code smell as well. Rather than passing around the "lang" param to multiple places, we should probably use a pub/sub instead (see `dispatch.js`).